templates/etc/apache2/sites-available/default.conf.j2

```diff
@@ -12,7 +12,8 @@
 
 {% if item.redirect_http|d() %}
     {{ apache__tpl_macros.get_redirect( item.redirect_http_code|d(307), "/", item.redirect_http) | indent(4) }}
-{% elif item.redirect_to_https|d(True) | bool %}
+{% elif item.redirect_to_https|default(apache__redirect_to_https) | bool %}
+    # Item's redirect: {{ item.redirect_to_https | default() }}   default: {{ apache__redirect_to_https }}
```

Comment:
I also like that but normally it should not be needed. But I think you are on to something and I am open to it but it should be optional and turned of by default. Then this would be a very handy feature I agree. Want to add `apache__template_debugging` boolean to make it happen?